syntax/text/parser/src/Luna/Syntax/Text/Parser/Hardcoded.hs

```diff
@@ -44,8 +44,10 @@ hardcodePrecRelMap = do
     Prec.writeRel EQ ("-"     :: Name) (Name.uminus :: Name)
     Prec.writeRel EQ ("%"     :: Name) ("*"         :: Name)
     Prec.writeRel EQ ("/"     :: Name) ("*"         :: Name)
+    Prec.writeRel EQ ("*"     :: Name) ("/"         :: Name)
```

Comment:
Can you please try to fix the real cause of this problem or just create a comment here that this is a hack? I prefer the former – it would be straightforward to add bidirectional equality there :)

Reply:
https://github.com/luna/luna/pull/396